experimental/displaynames/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+//! [`icu_displaynames`](crate) is one of the ['ICU4X`] components.
+//!
+//! TThis API provides a mapping of locale display names for regions.
+//!
+//! TODO: expand documentation
```

Comment:
Please add at least the boilerplate that this is experimental and that this is both a module in `icu` as well as its own crate. See `icu_segmenter`.

Reply:
Done! After this change, even if I run 'cargo make generate-readmes' first, I am getting an error that read-me file is out of sync with lib.rs.